deployment/6_register_modules_v1.js

```diff
@@ -61,8 +62,23 @@ const deploy = async (network, secret) => {
 
     for (let idx = 0; idx < wrappers.length; idx++) {
         let wrapper = wrappers[idx];
-        await multisigExecutor.executeCall(ModuleRegistryWrapper, "registerModule", [wrapper.contractAddress, utils.asciiToBytes32(wrapper._contract.contractName)]);
+        //await multisigExecutor.executeCall(ModuleRegistryWrapper, "registerModule", [wrapper.contractAddress, utils.asciiToBytes32(wrapper._contract.contractName)]);
```

Comment:
is this intentionally commented out?

Reply:
No, it's a left-over from the debug phase. I will uncomment.